src/main/resources/assets/css/main.less

```diff
@@ -141,11 +171,13 @@ a,
 }
 
 .breadcrumb {
-  background-color: #454a50;
+  padding: 15px;
+  margin: -20px 0 24px 0;
```

Comment:
Please only add necessary styles. Here `margin-top: -20px` should be enough.

Also, let's get rid of those rounded edges. Look at how Bootstrap defines them to override them.